code/modules/surgery/organs/organ_external.dm

```diff
@@ -781,14 +792,29 @@ Note that amputating the affected organ does in fact remove the infection from t
 	if(data["dna"])
 		sync_colour_to_dna()
 
+/obj/item/organ/external/proc/remove_embedded_object(obj/item/I)
+	embedded_objects -= I
+	UnregisterSignal(I, COMSIG_MOVABLE_MOVED)
+
+/obj/item/organ/external/proc/drop_embedded_objects()
+	var/turf/T = get_turf(src)
+	for(var/obj/item/I in embedded_objects)
+		remove_embedded_object(I)
+		forceMove(T)
+
+/obj/item/organ/external/proc/add_embedded_object(obj/item/I)
+	embedded_objects += I
+	I.forceMove(owner)
```

Comment:
Do you think we could just forceMove this into the organ itself? Code that deals with embedded objects seems to just check the `embedded_objects` list on the organ, and never looks inside the human itself. 

If this were to be changed, we can also remove the code from line 713 to 716 of the `/obj/item/organ/external/remove` proc.

Reply:
I don't see an issue with it myself. However, I believe that organs and such got removed from external organs their content. Though I do not remember the full reason for that.
@Fox-McCloud could steels suggestion be implemented without issues or?

Reply:
If I remember correctly [organs exist in nullspace while inside of a humanoid](https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/surgery/organs/organ_external.dm#L117). So, you wouldn't want to move the item inside the organ since that would mess up anything that relied on location or coordinates.